lib/widgets/home.dart

```diff
@@ -551,6 +553,23 @@ class _SwitchAccountButton extends _MenuButton {
   }
 }
 
+class _AboutZulipButton extends _MenuButton {
```

Comment:
nit in commit message:

    nav: Add an About Zulip button to the main menu
    Fixes: #1128

blank line between summary and body